### PR TITLE
Add edges to existing graph

### PR DESCRIPTION
**Pull request by siyuan0322**
Add edges to existing graph.

### CODE REVIEW

**Review by sighingnow — APPROVED** [review]
LGTM. Before merging, please test it against the graphscope repo to make sure everything works.

**Comment by siyuan0322** [comment]
@sighingnow Why has the coverage dropped so much?

**Comment by sighingnow** [comment]
codecov.io is a liar :)

File: modules/graph/fragment/arrow_fragment.h [hunk 1]
```diff
@@ -629,20 +630,440 @@ class ArrowFragment
...
+        vy_ie_lists[i].resize(extra_edge_label_num);
+        vy_ie_offsets_lists[i].resize(extra_edge_label_num);
+      }
+      vy_oe_lists[i].resize(extra_edge_label_num);
+      vy_oe_offsets_lists[i].resize(extra_edge_label_num);
+    }
+
+    // Make a copy of ovg2l map, since we need to add some extra outer vertices
+    // pulled in this fragment by new edges.
+    std::vector<ska::flat_hash_map<vid_t, vid_t>> ovg2l_maps(vertex_label_num_);
```
Comment (sighingnow) [inline modules/graph/fragment/arrow_fragment.h:713]: Use the hashmap builder rather than `ska::flat_hash_map<vid_t, vid_t>`.

Comment (siyuan0322) [reply modules/graph/fragment/arrow_fragment.h:713]: I reused the function `generate_outer_vertices_map` in line 768.

File: modules/graph/vertex_map/arrow_vertex_map.h [hunk 2]
```diff
@@ -308,6 +412,70 @@ class ArrowVertexMap<arrow::util::string_view, VID_T>
...
+          typename InternalType<oid_t>::vineyard_array_type>(
+          array_builder.Seal(client));
+      return Status::OK();
+    };
+
+    for (fid_t fid = 0; fid < fnum_; ++fid) {
+      for (label_id_t vlabel_id = 0; vlabel_id < extra_label_num; ++vlabel_id) {
+        tg.AddTask(builder_fn, fid, vlabel_id);
+      }
+    }
```
Comment (sighingnow) [inline modules/graph/vertex_map/arrow_vertex_map.h:446]: Do we need to build the whole vertexmap, rather than just for the newly added vertices?

**Pull request merged**